packages/app/src/systems/Transaction/components/ActivityList/ActivityListEmpty.tsx

```diff
@@ -0,0 +1,67 @@
+import { cssObj } from '@fuel-ui/css';
```

Comment:
I guess the illustration size is slighter different than Figma

<img width="375" alt="CleanShot 2023-01-09 at 20 10 29@2x" src="https://user-images.githubusercontent.com/2029172/211428245-6e4d6838-e464-472c-bf2c-092811e267a7.png">

Reply:
![Screenshot 2023-01-11 at 5 37 33 PM](https://user-images.githubusercontent.com/15849997/211878200-a4efd4a5-5356-4e9f-8665-ca771f365fb8.png)


Made it the right size.